analytics/cdk/emr-eks/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to your CDK TypeScript project
+
+This is a blank project for CDK development with TypeScript.
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+## Useful commands
+
+* `npm run build`   compile typescript to js
+* `npm run watch`   watch for changes and compile
+* `npm run test`    perform the jest unit tests
+* `cdk deploy`      deploy this stack to your default AWS account/region
+* `cdk diff`        compare deployed stack with current state
+* `cdk synth`       emits the synthesized CloudFormation template
```

Comment:
Readme is very light. I think it should tell the users what this does. e.g. what does it deploy? What permissions are created? What needs to be configured before deployed?

Reply:
This is the readme that comes out of the box with CDK, I am planning to add the readme as part of the github pages.